Winapp3/Winapp3.ini

```diff
@@ -261,6 +261,44 @@ ExcludeKey66=PATH|%WinDir%\SysWOW64\config\|SOFTWARE.bak*
 ExcludeKey67=PATH|%WinDir%\SysWOW64\config\|SOFTWARE.rhk.LOG*
 ExcludeKey68=PATH|%WinDir%\SysWOW64\config\systemprofile\|NTUSER.DAT.LOG*
 ExcludeKey69=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\DataSharing\Storage\|*.log
+ExcludeKey70=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\|*.tmp
+ExcludeKey71=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\DataSharing\Storage\|*.chk
+ExcludeKey72=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\Microsoft\Windows\WebCache\|*.chk
+ExcludeKey73=PATH|%WinDir%\Panther\|*.tmp
+ExcludeKey74=PATH|%ProgramFiles%\Adobe\Adobe Media Encoder CC 2019|*.bak
+ExcludeKey75=PATH|%LocalAppData%\Packages\Microsoft.Windows.Cortana_cw5n1h2txyewy\AC\AppCache\EGFPDUEB\1\|*.txt
+ExcludeKey76=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\en_GB\changelog.txt
+ExcludeKey77=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\es_ES\Changelog.txt
+ExcludeKey78=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\lv_LV\Changelog.txt
+ExcludeKey79=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\th_th\ChangeLog.txt

```

Comment:
For ExcludeKey76-79 you can use:
"%ProgramFiles%\\Adobe\\Adobe Photoshop CC \*\\Required\\Linguistics\\Providers\\Plugins2\\AdobeHunspellPlugin\\dictionaries\\\*\\|changelog.txt"